docker/Makefile

```diff
@@ -50,12 +56,12 @@ build: prep
 		--tag $(NAMESPACE)/puppetserver:$(version) \
 		puppetserver
 ifeq ($(IS_LATEST),true)
-	@docker tag $(NAMESPACE)/puppetserver-standalone:$(version) \
-		$(NAMESPACE)/puppetserver-standalone:latest
 	@docker tag $(NAMESPACE)/puppetserver:$(version) \
 		$(NAMESPACE)/puppetserver:latest
 endif
 
+build: build-standalone build-full
```

Comment:
FYI I verified that `build-standalone` does *not* get called twice during `make build`.

We could also just define this as `build: build-full` and then rely on `build-full: build-standalone` 🤷‍♂